ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/localzone/byo-subnet/ovn/mtu/day2/provision/cucushift-installer-rehearse-aws-ipi-localzone-byo-subnet-ovn-mtu-day2-provision-chain.yaml

```diff
@@ -15,7 +15,9 @@ chain:
   - ref: aws-provision-tags-for-byo-vpc
```

Comment:
Shall we need to move the step just like outpost provision chain?

Reply:
No. That's required by ALB tests.